k8s_references.go

```diff
@@ -156,3 +157,29 @@ func (ks kobjSlice) process() ([]interface{}, error) {
 	}
 	return objectRefs, nil
 }
+
+// ResourceRef is a full Kubernetes resource reference composed of Group, Version, Kind, Namespace and Name.
```

Comment:
Can you please check if the following suggested comment reads better?

```suggestion
// ResourceRef references a kubernetes object using Group, Version, Kind, Namespace and Name
```